src/context/ProductContext.jsx

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
+import {
+  createContext,
+  useContext,
+  useReducer,
+} from "react";
+import { useFetch } from "../custom_hooks/useAxios";
+
+const ProductContext = createContext();
+
+const ProductContextProvider = ({ children }) => {
+
+  // const fetchProduct = async () => {
+  //   try {
+  //     const res = await axios.get("api/products");
+  //     if (res.status === 200) {
+  //       setProductList(res.data.products);
+  //     }
+  //   } catch {
+  //     throw new Error("error");
+  //   }
+  // };
```

Comment:
Can remove commented code if not planning to use it in the future.

Reply:
yes, I have removed the code that is not needed some I need